src/views/Home.jsx

```diff
@@ -2,17 +2,22 @@ import './Home.css';
 import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
-import { getItemData, streamListItems } from '../api';
+import { getItemData, streamListItems, addItem } from '../api';
 
 export function Home({ setListToken }) {
 	const [errorMessage, setErrorMessage] = useState('');
 	const [joinListToken, setJoinListToken] = useState('');
 
 	const navigate = useNavigate();
 
+	function addPlaceholderItem(listId) {
+		addItem(listId, { itemName: '', daysUntilNextPurchase: 0 });
+	}
+
 	function makeNewList() {
 		const newToken = generateToken();
 		setListToken(newToken);
+		addPlaceholderItem(newToken);
```

Comment:
🔥 🔥 Making earlier ideas happen and giving the user more options for list sharing! And super readable.